guides/session.rst

```diff
@@ -80,42 +118,78 @@ With ``Behat\Mink\Session`` you can control your browsers cookies and headers:
     // retrieving response headers:
     print_r($session->getResponseHeaders());
 
-    // set cookie:
-    $session->setCookie('cookie name', 'value');
-
-    // get cookie:
-    echo $session->getCookie('cookie name');
-
-    // delete cookie:
-    $session->setCookie('cookie name', null);
-
 .. note::
 
     Headers handling is only supported in headless drivers, because there
     is no way browser controllers can get such information out of the browser.
 
 HTTP Authentication
-~~~~~~~~~~~~~~~~~~~
+-------------------
 
-Also, Mink session has a special method to perform HTTP Basic authentication:
+The Mink session has a special method to perform HTTP Basic authentication:
```

Comment:
Most of the previous sections start with `The session `. Maybe this one should start as well.

In general having all sections start with same word doesn't look too good. Maybe we should start each sentence with different word or something like that.